app/controllers/items_controller.rb

```diff
@@ -9,6 +9,7 @@ class ItemsController < ApplicationController
   before_action :forbid_modify, :only => [
     :add_collection, :set_collection, :remove_collection,
     :datastream_update,
+    :embargo_update,
```

Comment:
It's a catch-22.  If the authz is done within the action and fails, all the after actions run anyway.  If the authz is done as a before action, it can skip the action and all the after actions.  A consequence of using a before action is using the authz it uses.  So, it may be a question for @LynnMcRae as to whether the current forbid_modify action is reasonable for embargo updates.  It first checks for is_admin and then checks the object/APO for a role to `can_manage_content`.  So, yes, it's a functional change.  Without it, unauthorized requests on embargo update can trigger reindexing (unless bug).